man/systemd.exec.xml

```diff
@@ -482,175 +482,181 @@
 
       <varlistentry>
         <term><varname>StandardError=</varname></term>
-        <listitem><para>Controls where file descriptor 2 (STDERR) of
-        the executed processes is connected to. The available options
-        are identical to those of <varname>StandardOutput=</varname>,
-        with some exceptions: if set to <option>inherit</option> the
-        file descriptor used for standard output is duplicated for
-        standard error, while <option>fd</option> operates on the error
-        stream and will look by default for a descriptor named
+        <listitem><para>Controls where file descriptor 2 (STDERR) of the executed processes is connected to. The
+        available options are identical to those of <varname>StandardOutput=</varname>, with some exceptions: if set to
+        <option>inherit</option> the file descriptor used for standard output is duplicated for standard error, while
+        <option>fd:<replaceable>name</replaceable></option> will use a default file descriptor name of
         <literal>stderr</literal>.</para>
 
-        <para>This setting defaults to the value set with
-        <option>DefaultStandardError=</option> in
-        <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
-        which defaults to <option>inherit</option>. Note that setting
-        this parameter might result in additional dependencies to be
-        added to the unit (see above).</para></listitem>
+        <para>This setting defaults to the value set with <option>DefaultStandardError=</option> in
+        <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>, which
+        defaults to <option>inherit</option>. Note that setting this parameter might result in additional dependencies
+        to be added to the unit (see above).</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>StandardInputText=</varname></term>
+        <term><varname>StandardInputData=</varname></term>
+
+        <listitem><para>Configures arbitrary textual or binary data to pass via file descriptor 0 (STDIN) to the
+        executed processes. These settings have no effect unless <varname>StandardInput=</varname> is set to
+        <option>data</option>. Use this option to embedd process input data directly in the unit file.</para>
+
+        <para><varname>StandardInputText=</varname> accepts arbitrary textual data. C-style escapes for special
+        characters as well as the usual <literal>%</literal>-specifiers are resolved. Each time this setting is used a
+        new line is appended to the per-unit data buffer, followed by a newline character. Note that leading and
+        trailing whitespace of lines configured with this option is removed. If an empty line is specified the per-unit
+        data buffer is cleared (hence, in order to insert an empty line, add an additional <literal>\n</literal> to the
```

Comment:
I'd just say "If an empty line is specified, the list is reset (hence ...)", following the usual style.

Reply:
but it's not a list, but a buffer, and I think "clear" is more appropriate for buffers, than "reset"? dunno? anyway, i dropped the "per-unit data" bit now